metasploit-framework.gemspec

```diff
@@ -211,6 +211,6 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'eventmachine'
 
   # Earlier than latest Faraday gem is used to prevent upstream Octokit errors
-  spec.add_runtime_dependency 'faraday', '<= 0.17.0'
+  spec.add_runtime_dependency 'faraday', '0.17.3'
```

Comment:
```suggestion
  spec.add_runtime_dependency 'faraday'
```

Local testing suggests we can unlock this completely.  Octokit has released updates and `faraday 1.0.0` looks to be compatible.
https://github.com/lostisland/faraday/releases/tag/v1.0.0
https://github.com/lostisland/faraday/blob/master/UPGRADING.md

Basic search of all metasploit source and sub gems suggests no node changes are needed for `1.0.0`.

Reply:
Will test locally to ensure Faraday 1.0.0 works

Reply:
Works 👍